examples/example-app/project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.4.0")
+addSbtPlugin("org.virtuslab.ash" % "sbt-akka-serialization-helper" % "0.6.1")
```

Comment:
Nit: newline

Reply:
sure, thanks - just WIP - still much to be done from my side